site/source/_posts/2018/01/17-Announcing-Dojo-2-beta-5/Announcing-Dojo-2-beta-5.md

```diff
@@ -0,0 +1,119 @@
+
+The primary breaking change in `widget-core` is that [`HNode` was renamed to `VNode`](https://github.com/dojo/widget-core/pull/806)
+
+### New widgets
+
+Two new widgets were added in Beta 5:
+
+* [Progress](https://github.com/dojo/widgets/issues/385) (`@dojo/widgets/progress/Progress`)
+* [Toolbar](https://github.com/dojo/widgets/issues/386) (`@dojo/widgets/toolbar/Toolbar`)
+
+These widgets may be viewed in the [widget showcase](https://dojo.github.io/examples/widget-showcase/).
+
+### Stores
+
+A number of refinements were made since the initial `@dojo/stores` release in Beta 4:
+
+* [Add a basic `onChange` mechanism](https://github.com/dojo/stores/commit/b6165e6fd670874726e67ab166cb888967785d19)
+* Improvements to providing type definitions for [state](https://github.com/dojo/stores/pull/143) and [payload](https://github.com/dojo/stores/pull/152)
+
+See the [@dojo/stores readme](https://github.com/dojo/stores) for more information about this package.
+
+### Updated examples
+
+Our collection of [Dojo 2 examples](http://github.com/dojo/examples/) were updated for beta 5.
+
+In particular, the [kitchen sink version of ToDoMVC](https://github.com/dojo/examples/blob/master/todo-mvc-kitchensink) now uses `@dojo/stores`.
+
+Also the [Dojo 2 HNPWA](https://dojo-2-hnpwa-d668d.firebaseapp.com/) example was accepted and is now available.
+
+### ESM packages
+
+Dojo packages now ship ES modules alongside UMD! In the future we will be leveraging ESM for evergreen browser builds with webpack.
+
+### Theme creation
+
+A new [`cli-create-theme`](https://github.com/dojo/cli-create-theme) package has been added to make it easier to create a custom theme.
+
+### Migrate to TypeScript 2.6
+
+[TypeScript 2.6](https://blogs.msdn.microsoft.com/typescript/2017/10/31/announcing-typescript-2-6/) was released and introduced a number of significant improvements to the language, in particular the strictness of certain types of typings. These changes helped us further refine the type definitions and find a few bugs along the way. Dojo 2 beta 5 ships all of its packages leveraging TypeScript 2.6, and all packages conform to `strict` type mode.
+
+## What's next?
+
+We are nearly there for the initial 2.0 release. For RC1 in just a few weeks, you can look forward to:
+
+*  More end developer tooling improvements
+	* `dojo/cli-build-app` Finish re-architecture of the build system
+	* `dojo/cli-build-widget` Focused build process for Dojo 2 widgets as web components
+* `dojo/themes` A new package for Dojo 2 themes
+* `dojo/widget-core` Finish DOM abstraction
+* `dojo/widgets` Standardize widget events
+* Additional theme to provide Dojo 1 flat theme style interoperability
+
+At the moment, we are planning for RC1 in early February.
+
+We are also working on additional packages which may land in time for 2.0, but more likely for a subsequent release:
+
+* `dojo/devtool` A development tool for Dojo 2 (Chrome and Firefox)
```

Comment:
Do you think we should just say working on a solution for a Dojo 2 dev tool plugin and and dojo 2 powered grid? Basically the the devtool and diagnostics packages themselves are under review and somewhat in flux so although we will have a devtool, what that exactly looks like is currently unknown.

Also defo shouldn't say that these may land in time for 2.0 because that that's not going to happen :)